airbyte-commons/src/main/java/io/airbyte/commons/features/FeatureFlagHelper.java

```diff
@@ -4,31 +4,40 @@
 
 package io.airbyte.commons.features;
 
+import com.google.common.annotations.VisibleForTesting;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.UUID;
+import java.util.function.Function;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
 public class FeatureFlagHelper {
 
   public static boolean isFieldSelectionEnabledForWorkspace(final FeatureFlags featureFlags, final UUID workspaceId) {
-    final String workspaceIdsString = featureFlags.fieldSelectionWorkspaces();
+    return isWorkspaceIncludedInFlag(featureFlags, FeatureFlags::fieldSelectionWorkspaces, workspaceId) || featureFlags.applyFieldSelection();
+  }
+
+  public static boolean isStrictComparisonNormalizationEnabledForWorkspace(final FeatureFlags featureFlags, final UUID workspaceId) {
+    return isWorkspaceIncludedInFlag(featureFlags, FeatureFlags::strictComparisonNormalizationWorkspaces, workspaceId);
+  }
+
+  @VisibleForTesting
+  static boolean isWorkspaceIncludedInFlag(final FeatureFlags featureFlags,
+                                           final Function<FeatureFlags, String> flagRetriever,
+                                           final UUID workspaceId) {
+    final String workspaceIdsString = flagRetriever.apply(featureFlags);
     final Set<UUID> workspaceIds = new HashSet<>();
     if (workspaceIdsString != null && !workspaceIdsString.isEmpty()) {
       for (final String id : workspaceIdsString.split(",")) {
         try {
           workspaceIds.add(UUID.fromString(id));
         } catch (final IllegalArgumentException e) {
-          log.warn("Malformed workspace id for field selection: {}", id);
+          log.warn("Malformed workspace id: {}", id);
```

Comment:
Can we pass the context (field selection or strict comparison), I think that would be helpful if it came to debugging which var has an issue.

Reply:
done

Reply:
I think https://github.com/airbytehq/airbyte/pull/22381/commits/35ba27e702083f97895c7b744177a427bd8a90c6 accomplishes this, hopefully I didn't miss something